frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -941,6 +944,16 @@ function ReaderFooter:addToMainMenu(menu_items)
                     self.settings.lock_tap = not self.settings.lock_tap
                 end,
             },
+            {
+                text = _("Show SkimWidget by holding the footer"),
```

Comment:
Just "Skim on hold" ? "Long-press to skim" ? @Frenzie ?

Reply:
I was thinking something like this was not really user friendly, but it's fine by me. I'll wait for Frenzie to comment when he has some free time.

Reply:
"SkimWidget" isn't either. ;-)

I'd go with `Hold (footer) to skim`.

Reply:
I assume `(footer)` was meant not to be included literally, but as to allow @Galunid to choose between these 2 alternatives:
`Hold to skim` or `Hold footer to skim`
(I'm fine with any of them.)

Reply:
My bad, opened PR with fix